web-www/src/sections/case-studies/case-study/BottomSection.tsx

```diff
@@ -6,24 +6,7 @@ import clsx from 'clsx';
 
 import BackgroundSection from '../../../components/BackgroundSection';
 import Title from 'web-components/lib/components/title';
-
-interface BottomSectionProps {
-  quote: string;
-  person: {
-    name: string;
-    image: {
-      childImageSharp: {
-        fluid: FluidObject;
-      };
-    };
-    role: string;
-  };
-  background: {
-    childImageSharp: {
-      fluid: FluidObject;
-    };
-  };
-}
+import { SanityCaseStudyBottomSection } from '../../../generated/graphql';
```

Comment:
Basically the `Sanity` items are coming directly from the Sanity GraphQL server, and the others (which are probably the majority) are created by running the codegen and based on the `query` name.

So if I made a graphql 
```graphql
query myData {}
```

There would be a type generated from that called `MyDataQuery`. 

If it would be helpful, I could walk through this PR some over a call - I realize there's a lot going on